src/riak_ql_ddl.erl

```diff
@@ -1174,6 +1344,14 @@ is_query_valid_no_subexpressions_1_test() ->
                                   "AND myfamily = 'bob' ")
       ).
 
+is_insert_valid_null_test() ->
```

Comment:
I think this test (or a similar one) needs to check the resulting record; any unquoted strings are being interpreted as strings.